### DISCUSSION TITLE
需要一些建议：关于图床帮助的入口应该放的哪里

### DISCUSSION THREAD

**Discussion by z0z0r4** [LTCatt/PCL2Help#234] [General]
放在提交帮助内？
PCL2 帮助库编写规范/帮助内容规范内？

**Comment by z0z0r4** [comment]
https://github.com/z0z0r4/PCL2Help/blob/master/%E5%B8%AE%E5%8A%A9/%E6%8F%90%E4%BA%A4%E5%B8%AE%E5%8A%A9%20-%20%E7%BC%96%E5%86%99%E8%A7%84%E8%8C%83.xaml
当前图片帮助位置
![image](https://user-images.githubusercontent.com/78744121/180607436-c5749240-4b21-4a80-aa22-0cdf4d7cf450.png)

**Comment by WTP016-CN** [comment]
我觉得要不写个子页面然后加上使用教程放进去

**Comment by WTP016-CN** [comment]
稍微对你的文本做了一点更改 去 PR 看一下呗
还有就是帮助库里面网站不能带有图标
